tests/Jellyfin.Common.Tests/Extensions/StringExtensionsTests.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using MediaBrowser.Common.Extensions;
+using Xunit;
+
+namespace Jellyfin.Common.Tests.Extensions
+{
+    public class StringExtensionsTests
+    {
+        [Theory]
+        [InlineData("Banana split", ' ', "Banana")]
+        [InlineData("Banana split", 'q', "Banana split")]
```

Comment:
How about test cases for an empty string for the needle and/or haystack?

Reply:
You can't have an empty char

Reply:
But the haystack is a string that could be empty? Anyways nbd there can be more tests added later.